homeassistant/components/starline/lock.py

```diff
@@ -66,7 +68,7 @@ def icon(self):
         )
 
     @property
-    def is_locked(self):
+    def is_locked(self) -> bool | None:
```

Comment:
I wasn't 100% on this one, but it looks like `car_state` is a `dict[str, bool]` and that would make it return a nullable type.